src/main/java/com/faforever/client/replay/ReplayCardController.java

```diff
@@ -101,7 +101,9 @@ public void setReplay(Replay replay) {
     Integer replayTicks = replay.getReplayTicks();
     if (replayTicks != null) {
       durationLabel.setText(timeService.shortDuration(Duration.ofMillis(replayTicks * 100)));
-      ((Label) durationLabel.getGraphic()).setText("");
+      // FIXME which icon was added in https://github.com/FAForever/downlords-faf-client/commit/58357c603eafead218ef7cceb8907e86c5d864b6#r40460680
+//      durationLabel.getGraphic().getStyleClass().remove("duration-icon");
+//      durationLabel.getGraphic().getStyleClass().remove("time-icon");
```

Comment:
Not sure what u mean here. But I guess u need to find out what icon corresponds to this unicode. I guess if u dont, then leave the old code and add the fix me

Reply:
Did you look at the discussion https://github.com/FAForever/downlords-faf-client/commit/58357c603eafead218ef7cceb8907e86c5d864b6#r40460680? An icon was added, but it was done by directly modifying the font rather than adding it via icomoon. This icon is lost in current client version and I wasn't even able to look it up using the font from his commit. So whether or not I leave the old code, there is no icon.

Reply:
ok